ReactAndroid/src/main/java/com/facebook/react/views/viewpager/ReactViewPager.java

```diff
@@ -154,6 +158,7 @@ public boolean onTouchEvent(MotionEvent ev) {
     if (!mScrollEnabled) {
       return false;
     }
```

Comment:
If you are going to add a check to mScrollEnabled at the start of the method, delete this check here